config/locales/models/groups/en.yml

```diff
@@ -9,7 +9,10 @@ en:
       models:
         group:
           attributes:
+            group_name:
+              format: must only contain alphanumeric, hyphen, a blank space, or underscore
```

Comment:
For this one, I think it's actually a bug in the old message: add the attribute name here and for `repo_name` as well, to be consistent with the other messages